example/ios/Podfile

```
@@ -1,4 +1,4 @@
-require_relative '../node_modules/react-native-test-app/test_app'
+require_relative '../../node_modules/react-native-test-app/test_app'
```

The new template resolves RNTA instead of hard coding the path:
```suggestion
ws_dir = Pathname.new(__dir__)
ws_dir = ws_dir.parent until
  File.exist?("#{ws_dir}/node_modules/react-native-test-app/test_app.rb") ||
  ws_dir.expand_path.to_s == '/'
require "#{ws_dir}/node_modules/react-native-test-app/test_app.rb"
```

---

Right, I didn't bother with re-initializing the template this time around, but I guess I can update these since you so nicely made them suggestions :)